samples/nrf9160/http_application_update/Kconfig

```diff
@@ -16,8 +16,20 @@ config APPLICATION_VERSION
 	int "Application version"
 	default 1
 
+config ENABLE_HTTPS
+	bool "Enable HTTPS download"
+	depends on DOWNLOAD_CLIENT_TLS
+
+config HTTPS_SECURITY_TAG
+	int
+	prompt "HTTPS security tag" if ENABLE_HTTPS
```

Comment:
Any particular reason to define this config like this? Typically, for dependencies between configs we use `depends on` keywords. AFAIK, the current approach will create a hidden config in case HTTPS is disabled.

Reply:
The idea was to have it set to ` -1` when ENABLE_HTTPS is not set and then a value when it is enabled but I found out that it's not really possible. I'll fix it.